src/send_kcidb.py

```diff
@@ -190,21 +190,24 @@ def _parse_checkout_node(self, origin, checkout_node):
             }
         }]
 
-    def _get_output_files(self, artifacts: dict, exclude_properties=None):
-        output_files = []
+    def _filter_artifacts(self, artifacts: dict, exclude_properties=None, 
+                          include_properties=None):
+        input_files = []
```

Comment:
```suggestion
        filtered_artifacts = []
```

Or even `result`, since this method will be used by both output and input files

Reply:
Oh sorry, I completely missed that! Thanks!